boards/intel/btl/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Create an EFI image
+if(CONFIG_BUILD_OUTPUT_EFI)
+set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+  COMMAND ${PYTHON_EXECUTABLE} ${PROJECT_SOURCE_DIR}/arch/x86/zefi/zefi.py
```

Comment:
ZEPHYR_BASE?

Reply:
Explanation please

Reply:
`PROJECT_SOURCE_DIR` is the path from `project()`, it should be `ZEPHYR_BASE` since this is a zephyr file

Reply:
@nordicjm 

> ZEPHYR_BASE?

This type of feedback is very unhelpful, please spend some time explaining what you are asking for. Not everyone understands what you mean and why this might be a problem. 

https://docs.zephyrproject.org/latest/contribute/reviewer_expectations.html#:~:text=Reviewers%20shall%20be,request%20for%20changes.